src/v0/destinations/hs/util.js

```diff
@@ -506,6 +509,7 @@ const getExistingData = async (inputs, destination) => {
 
   while (checkAfter) {
     const endpoint = IDENTIFY_CRM_SEARCH_ALL_OBJECTS.replace(':objectType', objectType);
+    const endpointPath = `objects/:objectType/search`;
```

Comment:
`:objectType` won't this fill up the actual value ?
```suggestion
    const endpointPath = `objects/<objectType>/search`;
```

Reply:
No, it is just a placeholder